regulations/static/regulations/css/scss/comment/_comment-read.scss

```diff
@@ -106,10 +106,10 @@ Preamble Read Mode
       height: 10px;
       width: 10px;
       vertical-align: middle;
-      background-color: $gray_lighter;
+      background-color: $panel_background_color;
```

Comment:
This is slightly darker than the previous value -- would you mind running a quick contrast check to make sure it's okay?